virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -6,21 +6,30 @@ toc::[]
 
 Before you install {VirtProductName}, ensure that your {product-title} cluster meets the following requirements:
 
-* Your cluster must be installed on
-xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS (RHCOS) workers.
+* Your cluster must be installed using any kind of
+xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure installation method (Installer-provisioned, User-provisioned, or Assisted installer) with Red Hat Enterprise Linux CoreOS (RHCOS) workers.
 
 [NOTE]
 ====
 {VirtProductName} only supports RHCOS worker nodes. RHEL 7 or RHEL 8 nodes are not supported.
 ====
 
-* Additionally, your cluster must use xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[deploy machine health checks] to maintain high availability (HA) of virtual machines.
+* Additionally, there are two options to maintain virtual machines' high availability (HA):
```

Comment:
Suggestion: "There are two options to maintain high availability (HA) of virtual machines:"